lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -136,7 +137,7 @@ def _parse_search_results(self, response) -> Tuple[list, list]:
                 else:
                     raise Exception
             except Exception:
-                logger.warn(f"Parsing for result {entity_urn} failed")
+                logger.exception(f"Parsing for result {entity_urn} failed")
```

Comment:
do exceptions here get raised and break the ingestion?

Reply:
No, this is the same as the `logger.error` but it will include the stack trace in the log which is quite useful